lambdas/search/index.py

```diff
@@ -75,6 +75,8 @@ def lambda_handler(request):
         }
         size = 0
         _source = []
+        # Consider all documents when computing counts, etc.
+        terminate_after = None
```

Comment:
Are we sure that `terminate_after` is correct for `'search'`? If it terminates the search before going through the entire index, that would be very bad. If it's just a limit, that's different?